index.js

```diff
@@ -57,23 +57,24 @@ function derenderWords() {
 // after word has been input and it is correct change the color in the word bank
 function renderCorrectWord(wordID) {
     let correctWordElement = document.getElementById(wordID);
-    correctWordElement.setAttribute("class", "text-success fs-4");
+    correctWordElement.setAttribute("class", "text-success fs-4 px-2");
     game.correctWords += 1; // increment the number of correct words
     game.currentWord += 1; // move the word bank index by 1
 }
 
 // after a word has been input and it is wrong change the color in the word bank
 function renderErrorWord(wordID) {
     let errorWordElement = document.getElementById(wordID);
-    errorWordElement.setAttribute("class", "text-danger fs-4");
+    errorWordElement.setAttribute("class", "text-danger fs-4 px-2");
     game.errorWords += 1; // increment the number of wrond words
     game.currentWord += 1; // move the word bank index by 1
 }
 
 // render the current word in the word bank with a blue background so it stands out
 function renderCurrentWord(wordID) {
     let currentWordElement = document.getElementById(wordID);
-    currentWordElement.setAttribute("class", "bg-primary fs-4");
+    currentWordElement.setAttribute("class", "bg-primary-subtle fs-4 rounded-pill px-2");
+    currentWordElement.scrollIntoView({ behavior: "smooth", block: "center" });
```

Comment:
"Smooth" options is not working for me. It either needs a delay or be "instant"

I suggest that you make it "instant" for performance reasons.

Reply:
fixed this